src/webgpu/web_platform/reftests/ref/canvas_size_different_with_back_buffer_size-ref.html

```diff
@@ -0,0 +1,45 @@
+<html>
+  <title>WebGPU canvas_back_buffer_different_size (ref)</title>
+  <meta charset="utf-8" />
+  <link rel="help" href="https://gpuweb.github.io/gpuweb/" />
+  <canvas id="cvs_larger_than_back_buffer_ref" width="6" height="8"></canvas>
```

Comment:
To strictly match how we expect WebGPU to work, this canvas's inherent size should actually be 3x4, but its _CSS_ size should be 6px by 8px. `width=6 height=8 style="width:6px;height:8px;"`